packages/backend-modules/access/graphql/resolvers/_queries/accessGrantInfo.js

```diff
@@ -0,0 +1,23 @@
+const transformUser = require('@orbiting/backend-modules-auth/lib/transformUser')
+
+module.exports = async (_, { id }, { pgdb }) => {
+  const accessGrant = await pgdb.public.accessGrants.findOne({ id: id })
+  if (accessGrant) {
+    const granter = await pgdb.public.users.findOne({
+      id: accessGrant.granterUserId,
+    })
+
+    if (granter) {
+      const safeGranter = transformUser(granter, {
+        portraitUrl: granter.portraitUrl,
+      })
+
+      return {
+        granterName: safeGranter.name,
+        granterPortrait: safeGranter.portraitUrl,
+        message: accessGrant.message,
+      }
```

Comment:
@fuenkchen to match `user.portrait` we should also add `bw=true`, `resize=384x384` and `format=auto` to the url. 

`user.portrait` resolver:
https://github.com/republik/plattform/blob/6f438750505a595f4ce9a05b328d954a9e71f4c1/packages/backend-modules/republik/graphql/resolvers/User.js#L142-L166

(since it's done in the backend for `user.portrait` it would be odd to do it in the frontend – supporting `args.properties` on `granterPortrait` seems unnecessary here)